src/op_mode/show_ipsec_sa.py

```diff
@@ -14,119 +14,115 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import re
-import sys
+from hurry import filesize
+from re import split as re_split
+from sys import exit as sys_exit
```

Comment:
Please simply import it as exit to match all other files.

Reply:
Thanks! Fixed. Also, regrouped imports according to the PEP8.